app/controllers/billables_controller.rb

```diff
@@ -12,6 +12,12 @@ def bill_entries
         entry.update_attribute(:billed, true)
       end
     end
+    if params[:entries_to_unbill]
+      params[:entries_to_unbill].each do |entry_id|
+        entry = Entry.find(entry_id)
+        entry.update_attribute(:billed, false)
+      end
+    end
```

Comment:
Can we use [update_all](http://apidock.com/rails/ActiveRecord/Base/update_all/class) for this?